templates/textcat_prompt.jinja2

```diff
@@ -0,0 +1,11 @@
+From the text below, determine whether or not it contains a recipe for a
```

Comment:
It seems that this template is hard-coded to be about "recipes", is that intentional? While we could require the users to make their own template for their use-case, it seems more friendly to have a general prompt where they only need to provide us with the labels that they are interested in. 

The [ner example](https://github.com/explosion/prodigy-openai-recipes/blob/main/templates/ner_prompt.jinja2) gives a nice example of a template automatically generated out of labels passed as input. Would such a template be preferable here as well?

Reply:
I suppose this also opens up the question: is it the intent for this recipe to only support binary classification? That's certainly possible, just want to make sure it's the intent.

Reply:
Hi @koaning ! Thanks for the review! Yeah my original intention was to have it work on the recipe comments dataset (binary classification). But looking back I think it makes sense to have it cover non-exclusive classes. I'll update this PR!

Reply:
Grand. Let me know if you appreciate any help or when you've come around to making the template general. I'd like to give this recipe a spin 👍

Reply:
Nice! I'm only getting to it last Friday but I set myself a hard deadline tomorrow (Tuesday). If I'm still stuck by that day, I'll ping!

Reply:
@koaning I think this is ready for a spin! I updated the README for some examples you can try out!

Reply:
Did you push the readme changes? I did not see them in this PR.

Reply:
Oh sorry, I meant the PR description 😅